pathz/pathz.proto

```diff
@@ -72,60 +67,32 @@ service Pathz {
   //     not allowed and the attempt must fail proving that the OpenConfig
-  //     Path-based Authz Policy "works".
-  //     Once verfied, the client then proceeds to finalize the rotation.
-  //     If the new verification did not succeed the client will cancel the
+  //     gNMI Path-based Authorization Policy "works".
+  //     Once verified, the client then proceeds to finalize the rotation.
+  //     If the verification did not succeed the client will cancel the
   //     RPC thereby forcing the target to perform a rollback of the new
-  //     OpenConfig Path-based Authz Policy.
+  //     OpenConfig gNMI Path-based Authorization Policy.
   //
   //   Step 4: Final commit.
   //     Client ---> FinalizeRequest ----> Target
   //
-  rpc Rotate(stream RotatePathzRequest)
-      returns (stream RotatePathzResponse);
-
-  // Install will add a new OpenConfig Path-based Authz Policy on the target.
-  // If the stream is broken or any of the steps fail the target must rollback
-  // to the original state, i.e. revert any changes to the OpenConfig Path-based
-  // Authz Policy made during this RPC.
-  //
-  // Note that only one such RPC can be in progress. An attempt to call this
-  // RPC while another is already in progress will be rejected with the
-  // `UNAVAILABLE gRPC` error.
-  //
-  // An attempt to install a policy with already existing `id` will be rejected
-  // with the `ALREADY_EXISTS` gRPC error.
-  //
-  // As a policy has to be installed on a target before it is activated using
-  // the gNMI API it cannot be tested after uploading.
-  //
-  // The following describes the sequence of messages that must be exchanged
-  // in the Install() RPC.
-  //
-  // Sequence of expected messages:
-  //   Step 1: Start the stream
-  //     Client ----> Install() RPC stream begin ------> Target
-  //
-  //   Step 2: Send OpenConfig Path-based Authz Policy to Target.
-  //     Client --> UploadRequest(pathz_policy) ----> Target
-  //     Client <-- UploadResponse <--- Target
-  //
-  //   Step 3: Final commit.
-  //     Client ---> FinalizeRequest ----> Target
-  //
-  rpc Install(stream InstallPathzRequest)
-      returns (stream InstallPathzResponse);
+  rpc Rotate(stream RotateRequest)
+      returns (stream RotateResponse);
 
   // Probe allows for evaluation of the pathz policy engine response to a gNMI
   // operation performed by a user on a single gNMI path.
-  // The response is based on the currently active policy and is evaluated
-  // without actually performing the gNMI operation.
+  // The response is based on the instance of policy specified in the request
+  // and is evaluated without actually performing the gNMI operation.
   rpc Probe(ProbeRequest) returns (ProbeResponse);
+
+  // Get returns specified instance of the OpenConfig gNMI Path-based
+  // Authorization Policy together with its version and created-on information.
+  rpc Get(GetRequest) returns (GetRequest);
```

Comment:
Get(GetRequest) returns (GetResponse)

(you doubled the requests)

Reply:
:-) 
Changed